src/main/java/mateacademy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,125 @@
+package mateacademy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+
+import mateacademy.internetshop.dao.ItemDao;
+import mateacademy.internetshop.lib.Dao;
+import mateacademy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+    private static String DB_NAME = "test";
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public List<Item> getAll() {
+        return null;
+    }
+
+    @Override
+    public Item create(Item item) {
+        Statement stmt = null;
+        String query = "INSERT INTO " + DB_NAME + ".items (name, price) "
```

Comment:
we can use String.format() in all queries

Reply:
done